### DISCUSSION TITLE
I have a problem in vscode on kali linux

### DISCUSSION THREAD

**Discussion by mahmoudalfawair** [microsoft/vscode-cpptools#7776] [General]
![Screenshot from 2021-07-03 16-57-15](https://user-images.githubusercontent.com/84728207/124356651-ba24c180-dc1f-11eb-9fff-60628ce0b3b6.png)
and when i click f5 the respon is (launch: program '/home/mahmoud/Public/build/Debug/outdebug ' does not exist.

**Comment by sean-mcmanus** [comment]
compilerPath should be something like /usr/bin/gcc. Also, c_cpp_properties.json isn't used for building/debugging. See https://code.visualstudio.com/docs/cpp/config-linux
